docs/vault.md

```diff
@@ -85,7 +89,11 @@ Rotating the master key process also re-encrypts the keystore using the new mast
 
 ### Key rotation in replica sets
 
-Every `mongod` node in a replica set must have its own master key. The key rotation steps are the following:
+Starting with version 4.4.19-19, you can use the same master key for all replica set members. Vault assigns different versions to the master key stored at the same path. The `mongod` server identifies the master key by its version and the path and uses that identifier to read the correct master key during the restart.  
```

Comment:
I'd rewrite this line as follows:
```
Starting with version 4.4.19-19, the same secret path for all the nodes in the deployment can be used. Vault assigns different versions to the secrets stored at the same path. The `mongod` uses the `{<secret_path>, <secret_version>}` pair as an identifier of the master key and remembers that identifier internally so that the correct master key is retrieved from the Vault server during the restart.
```